packages/flutter/test/widgets/heroes_test.dart

```diff
@@ -1612,12 +1612,65 @@ Future<void> main() async {
     expect(tester.getTopLeft(find.byKey(firstKey)).dx, x0);
   });
 
-  testWidgets('Can override flight shuttle', (WidgetTester tester) async {
```

Comment:
we should keep the test as is unless it is testing the wrong behavior. we should add new test to cover the from hero instead.